src/lib/units/month.js

```diff
@@ -12,6 +12,7 @@ import isArray from '../utils/is-array';
 import indexOf from '../utils/index-of';
 import { createUTC } from '../create/utc';
 import getParsingFlags from '../create/parsing-flags';
+import isUndefined from '../utils/is-undefined';
 
```

Comment:
is this util being used? Note that my suggestion changed the meaning of the code. Previously you were checking for undefined, and now you're checking for falsiness. I think they achieve what amounts to the same thing, but I want to point out that semantic change.